### PR TITLE
chore: Add CLA and cla-bot config

### PR DESCRIPTION
**Pull request by bgins**
### Summary

This pull request makes the following changes:

- [x] Add CLA
- [x] Add notices file
- [x] Add cla-bot config

This pull request adds a contributor licensing agreement, a notices file to track those who have signed the CLA, and a [cla-bot](https://colineberhardt.github.io/cla-bot/#installing-cla-bot) configuration file. 

### Task/Issue reference

Implements: #173 

### Test plan

We have a test repo where you can preview signing a CLA: https://github.com/Lilypad-Tech/test-cla-bot.

### CODE REVIEW

File: .cla-bot [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "contributors": [
+
```
Comment (bgins) [inline .cla-bot:3]: Contributors will add their GitHub username here when they sign.

File: NOTICES.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Notices
+
+## License Acceptance
+
+Contributors may indicate their acceptance of the Contributor License Agreement by adding an entry with their full legal name and repository system identifier (i.e. GitHub username) to this document. The entry can be added through a pull request which must include a signed commit that adds the entry.
```
Comment (bgins) [inline NOTICES.md:5]: Entries like the following will be added below this line:

```

---------------------------------------------------------------------------------

Licensee’s name: Brian Ginsburg

Repository system identifier: bgins

---------------------------------------------------------------------------------
```

File: CLA.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+related patents, trademarks, and license agreements) of which you
+are personally aware, and conspicuously marking the work as
+"Submitted on behalf of a third-party: [named here]".
+
+8. You agree to notify Lilypad of any facts or circumstances of
+which you become aware that would make these representations
+inaccurate in any respect.
+
+This document was adapted from the Apache Software Foundation
+Individual Contributor License Agreement V2.2.
```
Comment (narbs91) [inline CLA.md:111]: Just as an FYI and I don't know if it matters but in the contributing doc in the other repo, the license is 2.0 but it's 2.2 here

**Review by narbs91 — APPROVED** [review]
LGTM

File: CLA.md (hunk 3 above)
Comment (bgins) [reply CLA.md:111]: Ah yeah, good detail to pick up on. There are two separate licenses, the software license and individual contributor license. They each have their own versions.

Comment (narbs91) [reply CLA.md:111]: Gotcha, makes sense

**Pull request merged**